src/Chapter14/Listing14.19.CustomAddAndRemoveHandlers.cs

```diff
@@ -11,27 +13,33 @@ public TemperatureArgs(float newTemperature)
 
             public float NewTemperature { get; set; }
         }
-
+     
         // Define the delegate data type
-        public delegate void TemperatureChangeHandler(
+        public delegate void EventHandler<TemperatureArgs>(
             object sender, TemperatureArgs newTemperature);
-
+        #endregion EXCLUDE
+        #region HIGHLIGHT
         // Define the event publisher
-        public event TemperatureChangeHandler OnTemperatureChange
+        public event EventHandler<TemperatureArgs> OnTemperatureChange
         {
             add
             {
-                _OnTemperatureChange = (TemperatureChangeHandler)System.Delegate.Combine(value, _OnTemperatureChange);
+                _OnTemperatureChange = 
+                    (EventHandler<TemperatureArgs>)System.Delegate.Combine
+                        (value, _OnTemperatureChange);
```

Comment:
I think this might want to be made a single line

Reply:
I changed it to be how it is written in the book, it's not a single line there